### PR TITLE
Not scheduling post if inputs are open

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Not scheduling post if inputs are open

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-61305

#### Release Note
```release-note
None
```

### CODE REVIEW

**Review by pvev — APPROVED** [review]
LGTM! 🚀

**Review by yasserfaraazkhan — APPROVED** [review]

File: webapp/channels/src/components/custom_status/date_time_input.tsx
Comment (larkox) [inline webapp/channels/src/components/custom_status/date_time_input.tsx:67]: Can you move this to one prop per line?

**Review by larkox — APPROVED** [review]
LGTM, just a minor nit.

File: webapp/channels/src/components/custom_status/date_time_input.tsx
Comment (harshilsharma63) [reply webapp/channels/src/components/custom_status/date_time_input.tsx]: Done.

**Comment by harshilsharma63** [comment]
This can be merged into master once the feature branch is merged into master as well

**Pull request merged**
